File: tourmatefe/types/bid.ts
import { TourGuide } from "./tour-guide"

export type Bid = {
    bidId: number,
    tourBidId: number,
    tourGuideId: number,
    amount: number,
    createdAt: string,
    comment?: string,
    status: string,
    tourGuide?: TourGuide
}

export type BidListResult = {
    bidId: number,
    tourBidId: number,
    tourGuideId: number,
    amount: number,
    createdAt: string,
    comment?: string,
    status: string,
    fullName: string,
    image: string,
}

File: TourMateBE/Repositories/Models/Bid.cs
using System;
using System.Collections.Generic;

namespace Repositories.Models;

public partial class Bid
{
    public int BidId { get; set; }

    public int TourBidId { get; set; }

    public int TourGuideId { get; set; }

    public float Amount { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Comment { get; set; }

    public string Status { get; set; } = null!;

    public virtual TourBid TourBid { get; set; } = null!;

    public virtual TourGuide TourGuide { get; set; } = null!;
}
